_lsb/008.md

```diff
@@ -129,9 +129,9 @@ Deriving the generated point $P$ at step 1:
 
 $\text{monero_apdu_generate_key_derivation}(P=x.G, henc(a))$
 
-$$= (8 x.a).G$$
-$$= (8x).a.G$$
-$$= 8x.A$$
+$$= henc((8 x.a).G)$$
+$$= henc((8x).a.G)$$
+$$= henc(8x.A)$$
```

Comment:
Would suggest the following to make clear what is part of the device/ app intermediate computation and what is visible to the outside :

```
The app computes :

$a \leftarrow henc(a)$

$8a.x.G = (8x).a.G$
$= 8x.A$

which equals $P$ by definition above. The output is then $henc(P)$, hence we learn a valid set $\{P, enc(P), hmac(P)\}$.
```

Does this look alright ?

Reply:
Yes, I modified the file and used an enumeration to make this easier to read. Thanks :)